modules/thoughtleadrBidAdapter.js

```diff
@@ -1,82 +1,171 @@
-'use strict';
-var bidfactory = require('src/bidfactory');
-var bidmanager = require('src/bidmanager');
-var utils = require('src/utils');
-var adloader_1 = require('src/adloader');
-var adaptermanager = require('src/adaptermanager');
-var ROOT_URL = '//cdn.thoughtleadr.com/v4/';
-var BID_AVAILABLE = 1;
-
-var ThoughtleadrAdapter = (function () {
-  function ThoughtleadrAdapter() {
+const bidfactory = require('src/bidfactory');
+const bidmanager = require('src/bidmanager');
+const utils = require('src/utils');
+const ajax_1 = require('src/ajax');
+const adaptermanager = require('src/adaptermanager');
+
+const COOKIE_SYNC_ID = 'tldr-cookie-sync-div';
+const UID_KEY = 'tldr_uid';
+const URL_API = 'tldr' in window && tldr.config.root_url ? tldr.config.root_url : '//a.thoughtleadr.com/v4/';
+const URL_CDN = 'tldr' in window && tldr.config.cdn_url ? tldr.config.cdn_url : '//cdn.thoughtleadr.com/v4/';
+const BID_AVAILABLE = 1;
+const BID_UNAVAILABLE = 2;
+
+function storageAvailable(type) {
+  try {
+    const storage = window[type],
+      x = '__storage_test__';
```

Comment:
Just had a lint update merged that's causing an error here:
`Split initialized 'const' declarations into multiple statements  one-var`
Once this is fixed we should be good to merge, thanks